password.go

```diff
@@ -88,22 +100,21 @@ type ArgonParams struct {
 	SaltSize    uint8
 }
 
-// Hash generates a argon2id hash of the input pass string with default settings
-// and returns the output in the specified string format and error value
-func Hash(pass string, customParams ...ArgonParams) (string, error) {
+// Hash generates a argon2id hash of the input pass string and returns the
+// output in the specified string format and error value.
+// Whether 'ArgonParams' is null, then it used default settings.
+func Hash(pass string, customParams *ArgonParams) (string, error) {
```

Comment:
I'm not sure this breaking change provides sufficient benefit, but it makes sense. Hmm.